exercises/concept/foto-fusionist/.docs/instructions.md

```diff
@@ -0,0 +1,70 @@
+# Instructions
+
+Your friend has gotten tired of paying for photo editing software and decided to write their own.
+Your friend decided to call the new software for Foto Fusionist.
+
+Foto Fusionist is a photo editing software that can do a lot of things, but you have only implemented a few of them so far.
+However, your friend is very happy with the progress and has asked you to implement a few more features.
+
+## 1. Convert binary representation to integer
+
+For the first feature, your friend wants to be able to convert a binary representation of a number to an integer.
+This is done because the software will be able to read the binary representation of a color channel and convert it to an integer.
+
+Implement the method `FotoFusionist.binary_to_int` that takes an argument `binary` of type `String`.
+The method should return the integer representation of the binary number.
+
+```crystal
+FotoFusionist.binary_to_int("10011")
+# => 19
+```
+
+## 2. Get least significant bit from integer
+
+The second feature your friend wants is to be able to get the [least significant bit][least-significant-bit] from an integer.
+The least significant bit is the bit that is the furthest to the right, it is called the least significant bit because it has the least value.
```

Comment:
```suggestion
The least significant bit is the bit that is the furthest to the right.
It is called the least significant bit because it has the least value.
```